lib/noticed/delivery_methods/action_push_native.rb

```diff
@@ -0,0 +1,23 @@
+module Noticed
+  module DeliveryMethods
+    class ActionPushNative < DeliveryMethod
+      required_options :devices, :format
+
+      def deliver
+        notification = notification_class
+          .with_apple(evaluate_option(:apple))
+          .with_google(evaluate_option(:google))
+          .with_data(evaluate_option(:data))
+          .new(evaluate_option(:format))
```

Comment:
should this be `.new(**evaluation_option(:format))` - I think I run into an error if not